TargetComposition/SymBuilder/HelpStrings.rc

```diff
@@ -15,10 +15,11 @@
 STRINGTABLE
 BEGIN
     SYMBOLBUILDER_IDS_MODULE_SYMBOLBUILDERSYMBOLS   "The symbol builder symbols for the module"
-    SYMBOLBUILDER_IDS_CREATESYMBOLS                 "CreateSymbols(module) - Creates symbol builder symbols for the module in question.  'module' can be the name or base address of a module or a module object"
+    SYMBOLBUILDER_IDS_CREATESYMBOLS                 "CreateSymbols(module, [options]) - Creates symbol builder symbols for the module in question.  'module' can be the name or base address of a module or a module object.  'options' is an object with properties that configures the symbols.  'options' currently allows .AutoImportSymbols = true/false"

```

Comment:
>'options' currently allows .AutoImportSymbols = true/false" [](http://example.com/codeflow?start=291&length=59)

Might be nice to describe what "AutoImportSymbols" actually does.

Reply:
Yep ...  I can add some verbiage in the help string...